package.json

```diff
@@ -70,21 +70,21 @@
     "bootstrap": "^4.3.1",
     "classnames": "^2.2.5",
     "color-hash": "^1.0.3",
-    "file-saver": "^1.3.3",
+    "file-saver": "^2.0.0",
     "lodash.debounce": "^4.0.8",
     "memoize-one": "^5.0.5",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
     "react-redux": "^7.1.0",
-    "react-transition-group": "^2.4.0",
+    "react-transition-group": "^4.0.0",
     "react-virtualized": "^9.21.1",
     "redux": "^4.0.0",
-    "redux-persist": "^5.10.0",
+    "redux-persist": "^5.0.0",
```

Comment:
Why the downgrade here?

Reply:
it installs the latest of version 5 anyways, but i changed it back